tests/117_untagged_tuple_variant.rs

```diff
@@ -23,7 +23,7 @@ fn test_ebkalderon_case() {
     flags: [
         "--enable-thing",
         "--enable-other-thing",
-        If("some-conditional", ["--enable-third-thing"]),
```

Comment:
Why did you remove this?

Reply:
I have no idea why this worked before, since we are testing untagged enums here. I think by adding the `If` it previously worked despite us missing support for untagged enums. However, now that it is properly implemented, you cannot give the name of the variant since untagged enums do not expect that